app/controllers/ChargeSummaryController.scala

```diff
@@ -40,6 +40,20 @@ import views.html.errorPages.CustomNotFoundError
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
+import scala.language.implicitConversions
+import controllers.ChargeSummaryController._
+
+object ChargeSummaryController {
+  case class ErrorCode(message: String, code: Int = 0, showInternalServerError: Boolean = true) extends RuntimeException(message)
+  type ItvcResponse[T]  = Either[ErrorCode, T]
+  type ItvcResult  = Future[Result]
+  implicit def toEither[T](t: T): ItvcResponse[T] = Right(t)
```

Comment:
I would suspect this is something we can cover by using cats library which is in scope of this project

Reply:
indeed, already investigated using Cats and we can use EitherT transformer for this type of conversion but didn't want to spend to much time perfecting it before getting some initial feedback, will do

Reply:
tried to use Cats, it didn't compile for some reason happy to pair with someone on this, but i don't favour invisible implicits that aren't on the screen for me to see, I prefer implicit type extensions that get called explicitly and are visible on the screen to give a hint to what's going on in the code, happy to discuss :)